sdk/workloads/Azure.ResourceManager.Workloads/src/Generated/Models/WordpressVersions.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Workloads.Models
+{
+    /// <summary> Application version. </summary>
+    public readonly partial struct WordpressVersions : IEquatable<WordpressVersions>
+    {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="WordpressVersions"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public WordpressVersions(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string Five43Value = "5.4.3";
+        private const string Five42Value = "5.4.2";
+        private const string Five41Value = "5.4.1";
+        private const string Five4Value = "5.4";
+
+        /// <summary> 5.4.3. </summary>
+        public static WordpressVersions Five43 { get; } = new WordpressVersions(Five43Value);
+        /// <summary> 5.4.2. </summary>
+        public static WordpressVersions Five42 { get; } = new WordpressVersions(Five42Value);
+        /// <summary> 5.4.1. </summary>
+        public static WordpressVersions Five41 { get; } = new WordpressVersions(Five41Value);
+        /// <summary> 5.4. </summary>
+        public static WordpressVersions Five4 { get; } = new WordpressVersions(Five4Value);
+        /// <summary> Determines if two <see cref="WordpressVersions"/> values are the same. </summary>
```

Comment:
These properties name looks weird, I would suggest using `Version5_4_0`, `Version5_4_1`, `Version5_4_2` and `Version5_4_3`, you can take a look at [this change](https://github.com/azure/azure-sdk-for-net/blob/main/sdk/cdn/Azure.ResourceManager.Cdn/src/Customization/Models/SslProtocol.cs#L15-L25) to see how to use customization to implement this.